### PR TITLE
feat: rename instances -> examples and make them work 

### PR DESCRIPTION
**Pull request by amritk**
- renamed instances to examples
- adding a new example works (though clicking on it does nothing... for now)
- simplified the schema file (requests.ts) based on zod best practices

### CODE REVIEW

File: packages/client-app/src/components/AddressBar/AddressBar.vue [hunk 1]
```diff
@@ -34,36 +29,36 @@ whenever(isMacOS() ? keys.meta_enter : keys.ctrl_enter, () =>
...
-      updateRequestInstance(
-        activeRequest.value.uid,
-        activeInstanceIdx,
-        toUpdate.key,
-        toUpdate.value,
-      )
-  },
-)
-
+// watch(
```
Comment (amritk) [inline packages/client-app/src/components/AddressBar/AddressBar.vue:32]: will bring this back later when we decide what it changes, ie tomorrow

File: packages/client-app/src/store/workspace.ts [hunk 2]
```diff
@@ -33,17 +33,18 @@ const requests = reactive<Record<string, RequestRef>>({})
...
   uid: string,
-  instanceIdx: number,
+  exampleUid: string,
   path: P,
-  value: PathValue<RequestInstance, P>,
-) {
-  requestMutators.edit(uid, `values.${instanceIdx}.${path}`, value)
+  value: PathValue<RequestExample, P>,
+) => {
+  // @ts-expect-error need Geoff to fix this plz!
```
Comment (amritk) [inline packages/client-app/src/store/workspace.ts:46]: help @geoffgscott

File: packages/oas-utils/src/helpers/iterateTitle.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Check for duplicate titles, and iterate title
+ */
+export const iterateTitle = (
```
Comment (amritk) [inline packages/oas-utils/src/helpers/iterateTitle.ts:4]: copied over from org

**Review by marclave — APPROVED** [review]
let's get this in, we can fast follow post review since it impacts so much!

**Pull request merged**